framework/deproxy_client.py

```diff
@@ -1,5 +1,6 @@
 import abc
 import time
```

Comment:
Well, what this code is a solution for https://github.com/tempesta-tech/tempesta-test/issues/115, a crucial task which blocked https://github.com/tempesta-tech/tempesta/issues/810#issuecomment-966690711 . Good work!